crates/store/re_entity_db/src/entity_db.rs

```diff
@@ -205,6 +205,11 @@ impl EntityDb {
         &self.data_store
     }
 
+    #[inline]
+    pub fn store_mut(&mut self) -> &mut ChunkStore {
+        &mut self.data_store
+    }
```

Comment:
I rather we did not do this, as it opens up the possibility of bugs like calling `entity_db.store_mut().insert_chunk(…)`.

See also:

https://github.com/rerun-io/rerun/blob/c19457d18bb48fca07f3c603a4e077e7e3598417/crates/store/re_entity_db/src/entity_db.rs#L27-L28